htdocs/core/class/html.form.class.php

```diff
@@ -7895,6 +7895,22 @@ public function showrefnav($object, $paramid, $morehtml = '', $shownav = 1, $fie
 			$fieldref = 'ref';
 		}
 
+		// Preparing gender's display if there is one
+		if ($object->gender) {
+			$addgendertxt = ' ';
+			switch ($object->gender) {
+				case 'man':
+					$addgendertxt .= '<i class="fas fa-mars"></i>';
+					break;
+				case 'woman':
+					$addgendertxt .= '<i class="fas fa-venus"></i>';
+					break;
+				case 'other':
+					$addgendertxt .= '<i class="fas fa-genderless"></i>';
+					break;
+			}
+		} else $addgendertxt = '';
```

Comment:
No need of else here, just add the init $addgendertxt = ''; before the if ($object->gender).
So, 
- you remove 1 else (it reduce the cyclomatic complexity score of code)
- It is clearer that var is correctly initialized, for all path, before being used.